libs/language-server/src/test/validation/checks/block-definition.spec.ts

```diff
@@ -0,0 +1,210 @@
+// SPDX-FileCopyrightText: 2023 Friedrich-Alexander-Universitat Erlangen-Nurnberg
+//
+// SPDX-License-Identifier: AGPL-3.0-only
+
+// eslint-disable-next-line @nrwl/nx/enforce-module-boundaries
+import { StdLangExtension } from '@jvalue/jayvee-extensions/std/lang';
```

Comment:
@georg-schwarz and I discussed whether to write a custom mock extension, but we did not consider this, maybe we should really write one.
But that would cause additional maintenance for the new extension

Reply:
I have created a test extension in language-server and removed all usages of StdExtension from language server.
The test extension blocks are also defined with tests in mind so they would not make any sense outside of a test environment.